core/ecschema-metadata/src/Context.ts

```diff
@@ -121,6 +121,17 @@ export class SchemaCache implements ISchemaLocater {
 
     return foundSchema as T;
   }
+
+  /**
+   * Iterates through each schema in _schema SchemaMap, then getItems for each Schema, and executes forEachFunction
+   * @param forEachFunction Function to be executed for each SchemaItem of Context
+   */
+  public iterateSchemaItems<T extends SchemaItem>(forEachFunction: (item: T) => void): void {

```

Comment:
Can you update this method to be more like the `Schema.getClasses` method?  This sets the method up as an [generator](https://basarat.gitbook.io/typescript/future-javascript/generators) that will allow you to loop over all items in the context and have them as a return to each iteration rather than having to pass in a function to it.

Reply:
Changed to use generator to iterate over context. A bit unsure as the usage syntax doesn't match example below.